### DISCUSSION TITLE
Bing maps Stuck

### DISCUSSION THREAD

**Discussion by rohanbatrain** [community/community#42500] [GitHub Education]
I am not able to proceed to select my school, it is available on google maps but is not on bing YET! 

![image](https://user-images.githubusercontent.com/116573125/209372214-b09235e7-e5c4-426e-a27f-d1957ac6d20e.png)

I can [access](https://goo.gl/maps/RbYeAoWHvphbKSkS7) it on google maps!

**Comment by jfullstackdev** [comment]
can you not use Google Chrome ? log in in Google so you can use it for the GitHub Education

**Reply by rohanbatrain** [reply]
I Think you are a bit confused! I mentioned Bing Maps specifically because that is what github uses on their portal, that has nothing to do with the browser. Also the website screenshot have been taken from firefox, and i tried the same on brave ( a chromium based browser) and microsoft edge is also a chromium based browser itself i have no idea what you are indicating towards.

**Reply by rohanbatrain** [reply]
I hope this additional information helps you understand that the problem is maps by Bing on github sign up portal for education platform. Here's an image for that as well. 

![bing](https://user-images.githubusercontent.com/116573125/209425133-e4940fcd-c392-425d-9ab7-71d6bfc333a6.png)

**Reply by rohanbatrain** [reply]
As you can see in the bottom left Microsoft Bing logo is displayed and this image has been extracted from github student enrollment page.

**Comment by NathanGlasby** [comment]
Hello! 

I’m having the same issue, expect my school is on Bing Maps, but it is weirdly not listed even though all other schools in my city are. My school is one of the biggest ones in the city. Please let me know if you ever find out why or get back an answer.

Thanks,
Nathan (@AteBotBo)

**Reply by rohanbatrain** [reply]
Don't expect anything dude, I have mailed official github and it's been a 9 days no reply.

**Comment by DikaMahardika** [comment]
i have the same problem,it is listed in google map and open street map but not on bing

**Comment by MahradH** [comment]
I have got same problem. The school is in map, but the list of schools is empty!

**Comment by Formzz** [comment]
same problem here
